src/Domain/Package/PackageManager.php

```diff
@@ -190,19 +190,20 @@ private function initializePackages(FixturePathHandler $fixture_path_handler, st
         continue;
       }
 
-      if ((array_key_exists('version', $datum) || array_key_exists('version_dev', $datum)) && $this->containsValidVersion($datum)) {
-        $this->addPackage($datum, $fixture_path_handler, $package_name);
-        continue;
-      }
-
-      if (isset($datum['core_matrix'])) {
+      if (array_key_exists('core_matrix', $datum)) {
         $constraints = array_values($datum['core_matrix']);
         foreach ($constraints as $constraint) {
           if ($this->containsValidVersion($constraint)) {
             $this->addPackage($datum, $fixture_path_handler, $package_name);
             break;
           }
         }
+        continue;
+      }
+
+      if (is_array($datum) && $this->containsValidVersion($datum)) {
+        $this->addPackage($datum, $fixture_path_handler, $package_name);
+        continue;
```

Comment:
I don't think this is necessary, is it? Since this is the last thing in the loop, it will continue either way, right?

Reply:
Good catch :)